plugins/devel/signals.cpp

```diff
@@ -0,0 +1,54 @@
+// This template is appropriate for plugins that simply provide one or more
+// commands, but don't need to be "enabled" to function.
+
+#include <string>
+#include <vector>
+#include <csignal>
+
+#include "Debug.h"
+#include "PluginManager.h"
+
+using std::string;
+using std::vector;
+
+using namespace DFHack;
+
+DFHACK_PLUGIN("signals");
+
+namespace DFHack {
+    DBG_DECLARE(signals, log, DebugCategory::LERROR);
+}
+
+static command_result send_signal(color_ostream &out, vector<string> &parameters);
+
+DFhackCExport command_result plugin_init(color_ostream &out, std::vector <PluginCommand> &commands) {
+    DEBUG(log, out).print("initializing %s\n", plugin_name);
+
+    commands.emplace_back(plugin_name, "", send_signal);
```

Comment:
I'll let others review the actual code, but I'll just add that since this doesn't have a help file, this is the only place where you can add command docs, and it should have some sort of docs to explain usage.

Reply:
You're saying to add the help/usage string?

Reply:
yeah, in lieu of a full .rst file. that's what the other devel plugins do.